ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -16,6 +16,22 @@
 ### Canonical Types  ###
 ########################
 
+DOAS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
```

Comment:
Please confim that there is no heating or cooling performed outsie of heat recovery

Reply:
@shambergoldstein - Heating and cooling only takes place in heatwheel within heat recovery